test/resources/layout.css

```diff
@@ -25,8 +25,10 @@ textarea.test-textarea {
     height: 100px;
     width: 100%;
     padding: 10px;
+    margin: 0;
     font-size: 0.9rem;
     border: solid 1px rgba(0,0,0,0.1);
+    font-size: 14px;
```

Comment:
I added these styles because I added textarea to the `un-normalize` file, but that affected all our code display text areas and made it hard to see what was going on.